pygenn/genn_groups.py

```diff
@@ -44,6 +45,48 @@ def set_var(self, var_name, values):
         """
         self.vars[var_name].set_values(values)
 
+    def pull_state_from_device(self):
+        """Wrapper around GeNNModel.pull_state_from_device"""
+        self._model.pull_state_from_device(self.name)
+
+    def pull_var_from_device(self, var_name):
+        """Wrapper around GeNNModel.pull_var_from_device
+
+        Args:
+        var_name    --  string with the name of the variable
+        """
+        self._model.pull_var_from_device(self.name, var_name)
+
+    def pull_extra_global_param_to_device(self, egp_name, size=1):
```

Comment:
Nitpick: pull_e_g_p_ *from* _device?

Reply:
Haha, looks like I copy-and-pasted [your typo](https://github.com/genn-team/genn/commit/4c440628b169203b10eac56b089c153208e5e80c#diff-608634e87bcf7f48cd21ee48051d988fR541-R547)

Reply:
whoops! xD